libs/template/presets/storefront/experience/pages/article-page.ts

```diff
@@ -2,11 +2,27 @@ import { StaticComponent } from '@spryker-oryx/experience';
 
 export const articlePage: StaticComponent = {
   type: 'Page',
-  meta: {
-    title: 'Article Page',
-    route: '/article/:id',
-    description: 'Default Article Page Description',
+  meta: { route: '/article/:id' },
```

Comment:
we should actually move those pages to labs package for the time being, as it's not yet part of final release.